abs.yaml

```diff
@@ -0,0 +1,6 @@
+build_parameters:
+  # 2024/3/7: We got an error on our CI but it's warniing on the WinDevBoxes:
+  # WARNING (gmp,Library/bin/libgmpxx-4.dll): .. but ['defaults/win-64::m2w64-gcc-libs-core==5.3.0=7'] not in reqs/run, 
+  # (i.e. it is overlinking) (likely) or a missing dependency (less likely)
+  # Seems 'gmp' doesn't really need 'm2w64-gcc-libs-core  # [win]' at runtime
```

Comment:
The error is not overdepending, it's overlinking, which is much worse. I'm worried to see that both concepts are mixed here. The error says that the package could fail at runtime.

Reply:
.But `libgmpxx-4.dll` isn't used by the downstream packages. We are in a stalemate.

Reply:
See the above comment https://github.com/AnacondaRecipes/gmp-feedstock/pull/4#discussion_r1521483888

Reply:
That's interesting. But I'm out of ideas on how to ship `gmp` with it

Reply:
@ifitchet Do you have any thoughts?

Reply:
I guess, if we were confident that `libgmpxx-4.dll` isn't used by any other R package and, as we are only providing `gmp` for the benefit of R, then we could remove it before packaging.  (Ideally, we would prevent it being built.)

If a customer comes along later desirous of `libgmpxx-n.dll` then someone has a _\*cough\*_ nice problem to solve.

Reply:
The error says to add `m2w64-gcc-libs-core` as a runtime dependency to solve the problem. (Unless I'm mistaken)

Reply:
I reply in that thread, see https://github.com/AnacondaRecipes/gmp-feedstock/pull/4#discussion_r1522037958. I'd like to test some things but I'm not sure if I need anything special to test or even where I can find `r-gmp`.

Reply:
@cbouss I've tried `export CFLAGS="${CFLAGS} -Wno-attributes -Wno-ignored-attributes -static-libstdc++"` but I do not see any differences in the result.

Reply:
Have you tried with LDFLAGS?

Reply:
The logs are the same with LDFLAGS:
```
Packaging gmp-6.3.0-h2bbff1b_0
number of files: 12
   INFO: sysroot: 'C:/Windows/' files: '['winhlp32.exe', 'win.ini', 'twain_32/wiatwain.ds', 'twain_32.dll']'
WARNING (gmp,Library/bin/libgmpxx-4.dll): Needed DSO Library/mingw-w64/bin/libgmp-10.dll found in ['defaults/win-64::m2w64-gmp==6.1.0=2']
WARNING (gmp,Library/bin/libgmpxx-4.dll): .. but ['defaults/win-64::m2w64-gmp==6.1.0=2'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
   INFO (gmp,Library/bin/libgmpxx-4.dll): Needed DSO C:/Windows/System32/KERNEL32.dll found in $SYSROOT
   INFO (gmp,Library/bin/libgmpxx-4.dll): Needed DSO C:/Windows/System32/msvcrt.dll found in $SYSROOT
WARNING (gmp,Library/bin/libgmpxx-4.dll): Needed DSO Library/mingw-w64/bin/libgcc_s_seh-1.dll found in ['defaults/win-64::m2w64-gcc-libs-core==5.3.0=7']
WARNING (gmp,Library/bin/libgmpxx-4.dll): .. but ['defaults/win-64::m2w64-gcc-libs-core==5.3.0=7'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
WARNING (gmp,Library/bin/libgmpxx-4.dll): Needed DSO Library/mingw-w64/bin/libstdc++-6.dll found in ['defaults/win-64::m2w64-gcc-libs==5.3.0=7']
WARNING (gmp,Library/bin/libgmpxx-4.dll): .. but ['defaults/win-64::m2w64-gcc-libs==5.3.0=7'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
   INFO (gmp,Library/bin/libgmp-10.dll): Needed DSO C:/Windows/System32/KERNEL32.dll found in $SYSROOT
   INFO (gmp,Library/bin/libgmp-10.dll): Needed DSO C:/Windows/System32/msvcrt.dll found in $SYSROOT
```